.NET/Microsoft.Recognizers.Definitions/English/DateTimeDefinitions.cs

```diff
@@ -541,5 +544,19 @@ public static class DateTimeDefinitions
 			{ "christmaseve", new string[] { "christmaseve" } },
 			{ "newyeareve", new string[] { "newyearseve", "newyeareve" } }
 		};
+		public static readonly Dictionary<string, int> WrittenDecades = new Dictionary<string, int>
+		{
+			{ "noughties", 0 },
+			{ "tens", 10 },
+			{ "twenties", 20 },
+			{ "two thousands", 20 },

```

Comment:
"two thousands" is 2000s, not 20s. This needs fixing.
Same for "noughties"
"twenty-tens" should also only covers 2010s.  Are you sure these cases cover only the right occurrences? I can't see that from the tests.

Reply:
Fixed and added some test cases.